<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSetRDRPropertyByExpression</class>
 <widget class="QDialog" name="DlgSetRDRPropertyByExpression">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set RDR Property by RegExp ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QVisibleHandleSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&lt;b&gt;set:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>saved settings:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QConfigComboboxWidget" name="widConfig" native="true"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <widget class="QWidget" name="scrollContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>343</width>
              <height>373</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnAdd">
             <property name="text">
              <string>add item</string>
             </property>
             <property name="icon">
              <iconset resource="../lib/lib.qrc">
               <normaloff>:/lib/file_add.png</normaloff>:/lib/file_add.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QListWidget" name="lstRDR">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnSelectAll">
           <property name="text">
            <string>select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelectNone">
           <property name="text">
            <string>deselect all</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnToggled">
           <property name="text">
            <string>toggle selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelSel">
           <property name="text">
            <string>select selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeselSel">
           <property name="text">
            <string>deselect selected</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;in these records:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QConfigComboboxWidget</class>
   <extends>QWidget</extends>
   <header>qconfigcomboboxwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVisibleHandleSplitter</class>
   <extends>QSplitter</extends>
   <header>qvisiblehandlesplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgSetRDRPropertyByExpression</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgSetRDRPropertyByExpression</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstRDR</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>DlgSetRDRPropertyByExpression</receiver>
   <slot>updateTest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>helpRequested()</signal>
   <receiver>DlgSetRDRPropertyByExpression</receiver>
   <slot>showHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>890</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateTest()</slot>
  <slot>showHelp()</slot>
 </slots>
</ui>
